uber/tests/models/test_getattr_fields.py

```diff
@@ -10,6 +10,7 @@ def test_ints():
 def test_label():
     assert '' == Attendee(paid=None).paid_label
     assert 'yes' == Attendee(paid=c.HAS_PAID).paid_label
+    assert 'none' == Attendee(paid="none").paid_label
```

Comment:
@kitsuta from this code here in models.py ```def _label()```:

```python
        try:
            val = int(val)
        except ValueError:
            return ''
```

in the case of the failing test, name is ```paid``` and val is ```none```. because of the int(val) it looks like the behavior is that if paid is not an int or None, _label() will return ''

So either that's wrong, or we should change the test to say:
```assert '' == Attendee(paid="none").paid_label```

which will fix it